### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by jonas-45**
The to-do-list branch has used webpack together with htmlwebpackplugin and file loader to build the to-do-list website with the following features;

1. The site has an input field to take in visitor's lists of to-do items
2. A function iterates over the to-do lists and is displayed dynamically on page load.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @jonas-45, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with an incomplete readme, your readme doesn't have all the necessary headers from the README file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    index: task2.index,
+    description: task2.desc,
+    completed: task2.completed,
+  },
+  {
+    index: task3.index,
+    description: task3.desc,
+    completed: task3.completed,
+  },
+];
```
Comment (uwadonat) [inline src/index.js:24]: - Please make sure you are implementing all [project requirements:](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md)
- [ ]  you still have hardcoded items in array. No add, edit and remove method implemented.
- [ ] Also make sure all changes to TO DO list are saved in local storage
- [ ] Please make sure your readme file is updated with all required features.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @jonas-45,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- No linter error ✔️
- Github flow was used ✔️
almost

## Required Changes ♻️
_Check the comments under the review. _

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @jonas-45 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding the key features section to your readme this will make your readme more descriptive 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jonas-45) [reply src/index.js:24]: I am going to work on it right away. Thanks

**Comment by jonas-45** [comment]
Thanks for your time and effort

**Pull request merged**